packages/i18n/src/create-i18n.js

```diff
@@ -186,12 +200,37 @@ export const createI18n = ( initialData, initialDomain, hooks ) => {
 		};
 	};
 
+	/**
+	 * @param {LocaleData} [data]
+	 * @param {string}     [domain]
+	 */
+	const doAddLocaleData = ( data, domain = 'default' ) => {
+		tannin.data[ domain ] = {
+			...DEFAULT_LOCALE_DATA,
```

Comment:
The `...DEFAULT_LOCALE_DATA` spread is redundant, because it has only the `''` key and that's gurananteed to be completely overwritten by the `''` key few lines later. The `doSetLocaleData` function has the same redundancy.